Readme.md

```diff
@@ -1,83 +1,34 @@
-# vscode
+# Prisma Language Support Test
```

Comment:
If this PR is to be reviewed and merged, the package name etc (descriptions below) needs to be changed.